core/src/nucleus/ribosome/api/get_links.rs

```diff
@@ -22,17 +28,14 @@ pub fn invoke_get_links(runtime: &mut Runtime, args: &RuntimeArgs) -> ZomeApiRes
         .unwrap()
         .dht()
         .get_links(input.entry_address, input.tag);
-    // Write result in wasm memory
-    match maybe_links {
-        Err(hc_err) => runtime.store_as_json(core_error!(hc_err)),
-        Ok(links) => {
-            let addresses = links
-                .iter()
-                .map(|eav| eav.value())
-                .collect::<Vec<Address>>();
-            runtime.store_as_json(GetLinksResult { addresses })
-        }
-    }
+
+    runtime.store_result(match maybe_links {
+        Ok(links) => Ok(links
+            .iter()
+            .map(|eav| eav.value())
+            .collect::<Vec<Address>>()),
```

Comment:
Whoops, yes we do need this back.  GetLinksResult needs to stay for when it returns more than just the addresses, i.e. the meta data

Reply:
kk, i'll put it all back in with a followup, my bad